integration-tests/jest/jest-utils.js

```diff
@@ -36,6 +36,11 @@ const registerTest = ({ fn, name, timeout = defaultTimeout, targetEnvironments =
     timeout)
 }
 
+const processResults = results => {
+  const errors = _.filter(_.isError, results)
+  return [errors, _.countBy('stack', errors)]
+}
+
```

Comment:
Could be

```suggestion
const processResults = _.flow(_.filter(_.isError), _.countBy('stack'))
```

and the call-site below can remain the same, thanks to Lodash FP currying :)

May also want to consider making the function name more specific (e.g. `groupByErrors`).

A comment around why you chose `'stack'` to count by may be helpful.

Reply:
I think this would just return the countBy result - but the logging requires both - the filtered error list and the grouped counts - I think this should stick around! But I can definitely add the comment about why "stack"